Core/FeatureFlag.swift

```diff
@@ -54,7 +54,11 @@ public enum FeatureFlag: String {
     case networkProtectionEnforceRoutes
 }
 
-extension FeatureFlag: FeatureFlagSourceProviding {
+extension FeatureFlag: FeatureFlagProtocol {
```

Comment:
Can we avoid calling protocols "Protocol" and call them something more in line with standard conventions please?